docs/spec/auth/scope.md

```diff
@@ -56,7 +56,7 @@ it.
 The resource name represent the name which identifies a resource for a resource
 provider. A resource is identified by this name and the provided resource type.
 An example of a resource name would be the name component of an image tag, such
-as "samalba/myapp".
+as "samalba/myapp" or "hostname/samalba/myapp". Note that the hostname is a more component that may have one `:` in it, due to a possible port number.
```

Comment:
I would put the last sentence down at the bottom. It only applies to "Resource Scope Grammar" and not resource names in general.